00-open-a-shapefile.Rmd

```diff
@@ -6,15 +6,17 @@ authors: [Joseph Stachelek, Leah Wasser]
 contributors: [Megan A. Jones, Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
-packagesLibraries: [raster, rgdal]
+packagesLibraries: [rgdal, raster]
 category: 
 tags: [module-1]
 mainTag: GIS-Spatial-Data
-description: "This lesson explains the how to open and plot point, line, and polygon shapefiles in R."
+description: "This lesson explains the how to open and plot point, line and
+polygon shapefiles in `R`."
 code1: 
 image:
   feature: NEONCarpentryHeader_2.png
-  credit: A collaboration between the National Ecological Observatory Network (NEON) and Data Carpentry
+  credit: A collaboration between the National Ecological Observatory Network
```

Comment:
I believe this will break the code. please remove the line in the credit text. YAML doesn't like multiple lines for indented list elements.